main.js

```diff
@@ -0,0 +1,83 @@
+let gameBoard = [["5", "4", "3", "2", "1"],
+[],
+[]];
+
+const board = {
+  printBoard() {
+    gameBoard.map(peg => {
```

Comment:
This is not a correct use of `map` - `map` should return from the callback as it is building a new array. This could be replaced with `forEach`.